package.json

```diff
@@ -12,37 +12,39 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.2.1",
-    "@angular/common": "^14.2.1",
-    "@angular/compiler": "^14.2.1",
-    "@angular/core": "^14.2.1",
-    "@angular/forms": "^14.2.1",
-    "@angular/platform-browser": "^14.2.1",
-    "@angular/platform-browser-dynamic": "^14.2.1",
-    "@angular/router": "^14.2.1",
-    "grapesjs": "^0.17.29",
-    "grapesjs-preset-newsletter": "^0.2.21",
-    "grapesjs-preset-webpage": "^0.1.11",
+    "@angular/animations": "^15.2.5",
+    "@angular/common": "^15.2.5",
+    "@angular/compiler": "^15.2.5",
+    "@angular/core": "^15.2.5",
+    "@angular/forms": "^15.2.5",
+    "@angular/platform-browser": "^15.2.5",
+    "@angular/platform-browser-dynamic": "^15.2.5",
+    "@angular/router": "^15.2.5",
+    "grapesjs": "^0.21.1",
+    "grapesjs-blocks-basic": "^1.0.1",
+    "grapesjs-preset-newsletter": "^1.0.1",
+    "grapesjs-preset-webpage": "^1.0.2",
     "rxjs": "~7.4.0",
     "tslib": "^2.3.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.2",
-    "@angular-eslint/builder": "14.0.4",
-    "@angular-eslint/eslint-plugin": "14.0.4",
-    "@angular-eslint/eslint-plugin-template": "14.0.4",
-    "@angular-eslint/schematics": "14.0.4",
-    "@angular-eslint/template-parser": "14.0.4",
-    "@angular/cli": "^14.2.2",
-    "@angular/compiler-cli": "^14.2.1",
+    "@angular-devkit/build-angular": "^15.2.4",
+    "@angular-eslint/builder": "15.2.1",
+    "@angular-eslint/eslint-plugin": "15.2.1",
+    "@angular-eslint/eslint-plugin-template": "15.2.1",
+    "@angular-eslint/schematics": "15.2.1",
+    "@angular-eslint/template-parser": "15.2.1",
+    "@angular/cli": "^15.2.4",
+    "@angular/compiler-cli": "^15.2.5",
     "@babel/eslint-plugin": "^7.16.5",
+    "@types/backbone": "^1.4.15",
```

Comment:
Where is this needed?

Reply:
I was under the impression that it was a dependency of the new GrapesJs, but quite possibly it was left over the test upgrades I had made in the earlier stages. I run some tests and new installations from scratch with it and the library and testing app runs fine it. Will remove it in the fixup.